README.md

```diff
@@ -16,23 +16,38 @@ Inspired heavily by [gperftools](https://code.google.com/p/gperftools/), and wri
 In your Gemfile add:
 
 ```ruby
-gem 'stackprof'
+gem "stackprof"
 ```
 
 Then run `$ bundle install`. Alternatively you can run `$ gem install stackprof`.
 
 
 ### Run
 
-in ruby:
+The `StackProf.run` method accepts a block with your code to be profiled.
 
 ``` ruby
-StackProf.run(mode: :cpu, out: 'tmp/stackprof-cpu-myapp.dump') do
+StackProf.run(mode: :cpu, out: "tmp/stackprof-cpu-myapp.dump") do
   #...
 end
 ```
 
-via rack:
+The `:out` option specifies where the sample data is written to.
+
+#### Mode
+
+When profiling a less complex code chunk it might be more helpful to switch from `:cpu` mode to `:object`. This is
+advisable when your `:cpu` run doesn't produce any samples.
+
+``` ruby
+StackProf.run(mode: :object, out: "tmp/stackprof-cpu-myapp.dump") do
+  #...
+end
+```
```

Comment:
I don't think this is accurate.  You should only switch from `:cpu` to `:object` when you want to change from profiling time to profiling object allocations.  If you run with `:cpu` doesn't produce any samples it means that your sampling frequency might not be high enough.  Measuring `:cpu` vs `:object` are completely different things (some code could take lots of time but allocate no objects, some code could run fast but allocate many objects).